src/openrct2/ride/Vehicle.cpp

```diff
@@ -1093,6 +1093,8 @@ void Vehicle::UpdateMeasurements()
         if (trackFlags & TRACK_ELEM_FLAG_HELIX)
         {
             uint8_t helixes = RideGetHelixSections(*curRide);
+            // helixes are stored on the last 5 bits of special_track_elements
+            // *DO NOT* change limits over 31 without revisiting this logic
```

Comment:
I think I’m just going to make this field private and have getters and setters, so that this isn’t a problem anymore.